_posts/2020-08-26-#19339.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: pr
+date: 2020-08-26
+title: "Redelgate AbsurdFee from Mempool to Clients"
+pr: 19339
+authors: [gzhao408]
+components: ["validation", "mempool", "rpc"]
+host: gzhao408
+status: upcoming
+commit: 67bb714
+---
+
+## Notes
+* [AcceptToMemoryPool()](https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp#L1084) (ATMP) processes a transaction for acceptance to the mempool. 
+In addition to enforcing consensus rules, it also applies the node’s policy. 
+[#19339](https://github.com/bitcoin/bitcoin/pull/19339) deals with acceptance logic that falls into neither category… and thus doesn’t belong in validation!
+
+* For reference, [Policy](https://github.com/bitcoin/bitcoin/tree/master/src/policy) = a node’s extra standards for validation, only applied to unconfirmed transactions. 
+It includes things like fees (e.g. [`-minrelaytxfee`](https://github.com/bitcoin/bitcoin/blob/3ab2582c7fe76d2839ab493512758d5601903c86/src/init.cpp#L547)) 
+and extra verification [flags](https://github.com/bitcoin/bitcoin/blob/3ab2582c7fe76d2839ab493512758d5601903c86/src/policy/policy.h#L56) (e.g. `SCRIPT_VERIFY_DISCOURAGE_UPGRADEABLE*`).
+This contrasts with consensus rules, which are universally applied in block validation.
+
+* On master, ATMP accepts an argument `const CAmount& nAbsurdFee`, and enforces it as a maximum fee amount; 
+this protects users from paying “absurdly high” fees resulting from an unintentional error. 
+It returns `TxValidationResult::TX_NOT_STANDARD` error for absurd fees, which seems to suggest that it is part of policy.
```

Comment:
s/returns /returns a /

Reply:
"It returns" and "it is part of policy" -> these *it*s are somewhat ambiguous and seem to be different things, perhaps write ATMP and `nAbsurdFee` (or what you are referring to here)